src/adv.py

```diff
@@ -38,6 +38,9 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player1 = Player(room['outside'])
+
+# print(player1.room.name)
 
 # Write a loop that:
 #
```

Comment:
Have you started your game loop yet? It doesn't seem like you pushed it up to Github if you have like you mentioned in standup yesterday.